pkg/labels/http_applicator.go

```diff
@@ -43,6 +43,10 @@ func (h *httpApplicator) SetLabel(labelType Type, id, name, value string) error
 	return util.Errorf("SetLabel not implemented for HttpApplicator (type %s, id %s, name %s, value %s)", labelType, id, name, value)
 }
 
+func (h *httpApplicator) SetLabels(labelType Type, id string, labels map[string]string) error {
+	return util.Errorf("SetLabel not implemented for HttpApplicator (type %s, id %s, labels %s)", labelType, id, labels)
```

Comment:
would have been best if that was `SetLabels`, but not too important I suppose.

I'm not sure you can `%s` a map though.